aspnetcore/fundamentals/dependency-injection/samples/6.x/WebApp1/Program.cs

```diff
@@ -0,0 +1,17 @@
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddScoped<IMyDependency, MyDependency>();
+
+var app = builder.Build();
+
+using (var serviceScope = app.Services.CreateScope())
+{
+    var services = serviceScope.ServiceProvider;
+
+    var myDependency = services.GetRequiredService<IMyDependency>();
+    myDependency.WriteMessage("Call services from main");
+}
+
+app.MapGet("/", () => "Hello World!");
+
+app.Run();
```

Comment:
> Glad you like the new hosting model! However, I think we might want to keep this section with app.Services instead of host.Services to show how to resolve a scoped services for a limited duration.

@halter73  how  is this?